grails-app/services/org/pih/warehouse/data/LocationDataService.groovy

```diff
@@ -144,8 +144,7 @@ class LocationDataService {
         }
         def locationType = null
         if (params.locationType) {
-            // Find by name with like, but value more the exact match
-            locationType = LocationType.findByNameOrNameIlike(params.locationType, params.locationType + "%")
+            locationType = LocationType.findByIdOrName(params.locationType, params.locationType)
```

Comment:
if we want to search by name also, don't we want to keep `"%"`? I see it being removed and can't guess the reason?

Reply:
nvm, it became `findByIdOrName` and was `NameIlike`